perses/samplers/samplers.py

```diff
@@ -363,6 +371,133 @@ def has_nan(self):
             return False
 
 ################################################################################
+# GHMC INTEGRATOR (TEMPORARY UNTIL MOVED TO OPENMMTOOLS)
```

Comment:
Isn't this [already](https://github.com/choderalab/openmmtools/blob/master/openmmtools/integrators.py#L527) in `openmoltools`? I guess that one needs to be fixed/have the `select` statement inserted?